samples/nrf9160/modem_shell/src/main.c

```diff
@@ -162,6 +166,10 @@ void main(void)
 	}
 #endif
 
+#if defined(CONFIG_MOSH_LOCATION)
+	/* Location library should be initialized between lte_lc_init and before normal mode */
```

Comment:
This comment is not mentioned in location.h. Should we add it there?

Reply:
I'll check if this is actually an obsolete comment.

Reply:
location_ctrl_init() is triggering subscribing for LTE_LC events. However, probably due to new LTE_LC event handling, it seems to be working (tested with cellular location) also if calling location_ctrl_init() before lte_lc_init() , and also after going to normal mode.
However, I think it is recommended that the init is done between those.

Reply:
I've currently modified this to the expected text and we'll do further modifications if tests show otherwise but we won't block the PR with this comment:
`/* Location library should be initialized before LTE normal mode */`

Reply:
I also tried placing location library init before lte_lc_init() and it seems to work. So current wording, i.e. without any reference to lte_lc_init(), seems correct to me.